### DISCUSSION TITLE
Mute Timings: Allow time intervals to follow Local Timezones or Daylight Savings

### DISCUSSION THREAD

**Discussion by jwood-krg** [grafana/grafana#45743] [Alerting]
I'd like to see a toggle or config flag to allow the time interval on a Mute Timing to follow DST. While UTC is a sensible default, many of our users would like their Mute Timings to account for daylight savings. As it stands, they will have to adjust all their Mute Timing configurations by hand every six months to offset the interval from UTC.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82988. Feel free to continue the discussion around this there. Thank you!
